app/views/dashboard/_activity.html.erb

```diff
@@ -97,16 +84,14 @@
 
   <div class="row">
     <% activity.each_with_index do |node, i| %>
-      <% if node.is_a?(Revision) || node.type == 'page' %>
-        <%= render partial: "dashboard/node_wiki",     locals: { node: node, index: i } %>
```

Comment:
We are going to need to add this back in, I am getting errors when I run this:

![FireShot Capture 251 - Action Controller_ Exception caught - localhost](https://user-images.githubusercontent.com/49460529/73457860-54554b80-4342-11ea-9301-26325fa19829.png)